2_munge/src/1_PlotPrep_and_Summary.R

```diff
@@ -0,0 +1,59 @@
+##This script munges our Mendota data to prep it for Figure 1 and then
+##outputs a text summary of the results
+
+library(dplyr)
+library(readr)
+library(stringr)
+library(whisker)
+library(readr)
+
+## Function for plotting the data and comparing models
+munge_data <- function(fileIn, project_output_dir, name){
+  
+  if(dir.exists(project_output_dir) == F){
+    dir.create(project_output_dir)
+  }
+  
+  evals <- readr::read_csv(fileIn, col_types = 'iccd') %>%
+    filter(str_detect(exper_id, 'similar_[0-9]+')) %>%
+    mutate(col = case_when(
+      model_type == 'pb' ~ '#1b9e77',
+      model_type == 'dl' ~'#d95f02',
+      model_type == 'pgdl' ~ '#7570b3'
+    ), pch = case_when(
+      model_type == 'pb' ~ 21,
+      model_type == 'dl' ~ 22,
+      model_type == 'pgdl' ~ 23
+    ), n_prof = as.numeric(str_extract(exper_id, '[0-9]+')))
+  
+  readr::write_csv(evals, file = file.path(project_output_dir, name))
+  print(paste0("Munged eval data has been saved to ", file.path(project_output_dir, name)))
+  return(evals)
```

Comment:
I see why you did it, but this approach of both writing an output file and returning the data as an R object violates the "1:1 function-to-target pairing" recommended in the PR instructions. It'd be best for a **targets** pipeline to choose just one approach - either write a file or return an object.